pkg/auth/routes/handler.go

```diff
@@ -0,0 +1,94 @@
+package routes
+
+import (
+	"context"
+	"encoding/json"
+	"monorepa/model"
+	"monorepa/service/auth"
+	"net/http"
+
+	"github.com/gorilla/mux"
+)
+
+type HandlerItemsServ struct {
+	router   *mux.Router
+	services *auth.MockAuthService
+	ctx      context.Context
+}
+
+func New(ctx context.Context, router *mux.Router) *HandlerItemsServ {
+	return &HandlerItemsServ{
+		router:   router,
+		services: &auth.MockAuthService{},
+		ctx:      ctx,
+	}
+}
+
+func (h *HandlerItemsServ) HandlerItems() {
+	h.router.HandleFunc("/login", h.GetJWTToken).Methods("GET")
+	h.router.HandleFunc("/get-cert/{version}", h.GetCertKey).Methods("GET")
+}
+
+func (h *HandlerItemsServ) GetJWTToken(w http.ResponseWriter, r *http.Request) {
+	var user model.User
+	err := json.NewDecoder(r.Body).Decode(&user)
+	if err != nil {
+		w.WriteHeader(http.StatusBadRequest)
+		return
+	}
+
+	token, err := h.services.Login(user)
+	if err != nil {
+		/**
+		 * change to global err
+		 */
+		if err.Error() == "wrong password" {
+			w.WriteHeader(http.StatusBadRequest)
+			return
+		}
+
+		w.WriteHeader(http.StatusInternalServerError)
+		return
+	}
+
+	w.Header().Set("token", token)
+	w.WriteHeader(http.StatusCreated)
```

Comment:
status "created" is for entities, i guess not for tokens. I would rather return 200 OK

Reply:
I create new JWT token based on user information. It seems to me that this thing is better to consider as entity.